cns/kubecontroller/nodenetworkconfig/metrics.go

```diff
@@ -24,12 +24,26 @@ var (
 			Help: "Unused IP count.",
 		},
 	)
+	hasNNC = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "nnc_has_nodenetworkconfig",
+			Help: "Has received a NodeNetworkConfig",
+		},
+	)
+	ncs = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "nnc_ncs",
```

Comment:
Typically, don't prometheus metrics that are counts end with  `_total`?

Reply:
Yes, but these are not counts in the way that Prometheus uses that term.

They say add `_total`
> for a unit-less accumulating count

This is an instantaneous count and may go up or down, ie not accumulating. I don't think total is appropriate. You wouldn't dx/dt (nnc_ncs) to get a rate, for example.

Their general guidance is to suffix the metric with the plural units - NC is the unit here, as is IP in the ones above.

Reply:
ah, okay. I understand now